.travis.yml

```diff
@@ -27,30 +34,18 @@ jobs:
       script: sbt +jacoco
 
     - stage: test
-    - script: sbt +publishLocal +plugin/test +plugin/scripted
+    - script: sbt +publishLocal +plugin/test
       name: "Test plugins JDK 8"
       env: TRAVIS_JDK=8
-    - script: sbt +publishLocal +plugin/test +plugin/scripted
+    - script: sbt +publishLocal +plugin/test
       name: "Test plugins JDK11"
       env: TRAVIS_JDK=11
-    - script: cd docs && sbt validateDocs evaluateSbtFiles
-      name: "Validate docs, Evaluate sbt files JDK 8"
-      env: TRAVIS_JDK=8
-    - script: cd docs && sbt validateDocs evaluateSbtFiles
-      name: "Validate docs, Evaluate sbt files JDK 11"
-      env: TRAVIS_JDK=11
 
     - stage: deploy
-      name: "TEMP Publish artifacts to Bintray"
-      script: sbt +mimaReportBinaryIssues
```

Comment:
This is strange. It was not publising at all, but checking mima. 

In any case, better to have the mima check on a new stage, before the test stage.

Reply:
Done